src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,298 @@
+            a[1U] = v;
+            ret[jss::FIELDS][i++] = a;
+        }
+
+        {
+            Json::Value a = Json::arrayValue;
+            a[0U] = "ObjectEndMarker";
+            Json::Value v = Json::objectValue;
+            v[jss::nth] = 1;
+            v[jss::isVLEncoded] = false;
+            v[jss::isSerialized] = true;
+            v[jss::isSigningField] = true;
+            v[jss::type] = "STObject";
+            a[1U] = v;
+            ret[jss::FIELDS][i++] = a;
+        }
+
+        {
+            Json::Value a = Json::arrayValue;
+            a[0U] = "ArrayEndMarker";
+            Json::Value v = Json::objectValue;
+            v[jss::nth] = 1;
+            v[jss::isVLEncoded] = false;
+            v[jss::isSerialized] = true;
+            v[jss::isSigningField] = true;
+            v[jss::type] = "STArray";
+            a[1U] = v;
+            ret[jss::FIELDS][i++] = a;
+        }
+
+        {
+            Json::Value a = Json::arrayValue;
+            a[0U] = "taker_gets_funded";
+            Json::Value v = Json::objectValue;
+            v[jss::nth] = 258;
+            v[jss::isVLEncoded] = false;
+            v[jss::isSerialized] = false;
+            v[jss::isSigningField] = false;
+            v[jss::type] = "Amount";
+            a[1U] = v;
+            ret[jss::FIELDS][i++] = a;
+        }
+
+        {
+            Json::Value a = Json::arrayValue;
+            a[0U] = "taker_pays_funded";
+            Json::Value v = Json::objectValue;
+            v[jss::nth] = 259;
+            v[jss::isVLEncoded] = false;
+            v[jss::isSerialized] = false;
+            v[jss::isSigningField] = false;
+            v[jss::type] = "Amount";
+            a[1U] = v;
+            ret[jss::FIELDS][i++] = a;
+        }
+
+        for (auto const& [code, f] : ripple::SField::knownCodeToField)
+        {
+            if (f->fieldName == "")
+                continue;
+
+            Json::Value innerObj = Json::objectValue;
+
+            uint32_t type = f->fieldType;
+
+            innerObj[jss::nth] = f->fieldValue;
+
+            innerObj[jss::isVLEncoded] =
+                (type == 7U /* Blob       */ || type == 8U /* AccountID  */ ||
+                 type == 19U /* Vector256  */);
+
+            innerObj[jss::isSerialized] =
+                (type < 10000 && f->fieldName != "hash" &&
+                 f->fieldName !=
+                     "index"); /* hash, index, TRANSACTION, LEDGER_ENTRY,
+                                  VALIDATION, METADATA */
+
+            innerObj[jss::isSigningField] = f->shouldInclude(false);
+
+            innerObj[jss::type] = type_map[type];
+
+            Json::Value innerArray = Json::arrayValue;
+            innerArray[0U] = f->fieldName;
+            innerArray[1U] = innerObj;
+
+            ret[jss::FIELDS][i++] = innerArray;
+        }
+
+        ret[jss::TRANSACTION_RESULTS] = Json::objectValue;
+
+        for (const auto& pair : detail::transResults())
```

Comment:
We shouldn't use the `detail` namespace from another module. (remove the function from the detail namespace).

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/d558b665cd2658b8fd6d2176662758f9efd8ae3f